browsertest/browsertest.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Browser tests</title>
+    <link rel="stylesheet" media="all" href="vendor/mocha.css">
+</head>
+<body>
+
+<div id="mocha"><p><a href=".">Index</a></p></div>
+<div id="messages"></div>
+<div id="fixtures"></div>
+
+<script src="vendor/mocha.js"></script>
+<script src="vendor/chai.js"></script>
+<script src="vendor/should.js"></script>
```

Comment:
I would prefer if we can include these vendor dependencies via some CDN. Then remove the included source from this PR.